libraries/helpers.rb

```diff
@@ -0,0 +1,38 @@
+module Chocolatey
+  module Helpers
+    # Get the ChocolateyInstall directory from the environment.
+    def chocolatey_install
+      ENV.fetch('ChocolateyInstall') { |env_var| machine_env_var(env_var) }
+    end
+
+    # The Chocolatey command.
+    #
+    # Reference: https://github.com/chocolatey/chocolatey-cookbook/pull/16#issuecomment-47975896
+    def chocolatey_executable
```

Comment:
Maybe we just need better comments here to explain why we are not using choco.exe and using the odd environment variable checking.